zvariant/src/signature.rs

```diff
@@ -511,6 +511,47 @@ impl std::fmt::Display for OwnedSignature {
     }
 }
 
+/// [`Signature`] that identifies a complete type.
```

Comment:
Let's keep this new type in a separate mod.

Reply:
ok